WordPress/Docs/DateTime/RestrictedFunctionsStandard.xml

```diff
@@ -0,0 +1,44 @@
+<documentation title="Restricted Date and Time Functions">
+    <standard>
+        <![CDATA[
+      The restricted functions date_default_timezone_set() and date() should not be used.
+
+      Using date_default_timezone_set() isn't allowed, because WordPress core needs default time zone set to UTC. Use DateTime object or WP core API functions instead.
+
+      Using date() isn't allowed, as it is affected by runtime timezone changes which can cause date/time to be incorrectly displayed. Use gmdate() instead.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: Using DateTime object.">
+            <![CDATA[
+$date = new DateTime();
+$date->setTimezone(
+    new DateTimeZone( 'Europe/Amsterdam' )
```

Comment:
Would it be an idea to "WordPressify" this code sample a little more and use something like `get_option()` with `gmt_offset` or `timezone_string` to retrieve the timezone to be passed to `new DateTimeZone()` ?

Reply:
I think the basic PHP example is fine there, people tend to change default time zone to something that is _not_ normal WP time zone.

Again, there is a problem of choice between a technical 1:1 code equivalent and idiomatic rewrite, that is hard to decide on for brief out-of-context example.